yb-voyager/cmd/exportData.go

```diff
@@ -293,6 +293,9 @@ func exportPGSnapshotWithPGdump(ctx context.Context, cancel context.CancelFunc,
 	if err != nil {
 		return fmt.Errorf("export snapshot: failed to create replication slot: %v", err)
 	}
+	utils.PrintAndLog(color.YellowString(fmt.Sprintf("Created replication slot '%s' on source PG database. "+
+		"Be sure to run 'end migration' command after completing/aborting this migration to drop the replication slot. "+
```

Comment:
What about the `cutover to target` in the msg?

Reply:
Oh just saw the note in description. 👍

Reply:
@makalaaneesh,  this might give the user the impression that the slot is open till the end migration is done if we don't tell them that this is also removed in the cutover phase.

Reply:
@priyanshi-yb True! I just thought it would be simpler this way, instead of getting into more details. But I suppose we can just say "Be sure to run cutover to target/end migration command after completing/aborting this migration ...."